packages/state-transition/src/cache/pubkeyCache.ts

```diff
@@ -52,6 +63,8 @@ export class PubkeyIndexMap {
  * Mutates `pubkey2index` and `index2pubkey`
  *
  * If pubkey caches are empty: SLOW CODE - 🐢
+ *
+ * TODO: Deal with this
```

Comment:
```suggestion
 * TODO EIP6110: Deal with this
```
Explain a bit more what's the problem here, what we have to deal with

Reply:
ping

Reply:
> ping

Was a mistake. No change needs to be made here